datadog_checks_dev/datadog_checks/dev/tooling/commands/create.py

```diff
@@ -104,7 +104,17 @@ def create(ctx, name, integration_type, location, non_interactive, quiet, dry_ru
     root = resolve_path(location) if location else get_root()
     path_sep = os.path.sep
 
-    integration_dir = os.path.join(root, normalize_package_name(name))
+    if integration_type == 'snmp_tile' and name.startswith('snmp_'):
+        name_without_snmp_prefix = name[len('snmp_'):]
+        echo_warning(f'Creating snmp_tile for vendor ({name}) in directory (snmp_{name})')
```

Comment:
Question: why not just ask for vendor name (e.g. "My Vendor", display name style) as input instead of snmp_<vendor> and then ask for user input ?

Reply:
It is possible, would require to change `name` from argument to option, it would impact other template types

Reply:
My bad, misread the code 👍